app/Http/Controllers/CategoryController.php

```diff
@@ -9,8 +9,7 @@ class CategoryController extends Controller
 {
     public function index()
     {
-        $category = new Category;
-        $categories = $category->getCategories(5);
+        $categories = Category::orderBy('id', 'DESC')->paginate(5);
```

Comment:
alway remember the pattern of MVC, orderBy is model related function, make a function in model category that will do this behavior and the controller will just call that function.